Gopkg.lock

```diff
@@ -1,6 +1,47 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```

Comment:
nit: update the commit to `vendor: add azure sdk and cluster-api-provider-azure dependencies `

Reply:
fixed